src/main/java/org/sagebionetworks/bridge/reporter/helper/BridgeHelper.java

```diff
@@ -26,6 +28,8 @@
 public class BridgeHelper {
     private static final Logger LOG = LoggerFactory.getLogger(BridgeHelper.class);
 
+    static final long MAX_PAGE_SIZE = 100L;
```

Comment:
Max page size 100 is very large. Do we need to paginate through our uploads this fast? Can we paginate at a more reasonable rate, like 10 or so?